newrelic-agent/src/main/java/com/newrelic/agent/instrumentation/weaver/ClassWeaverService.java

```diff
@@ -218,17 +218,16 @@ private WeavePackageConfig createWeavePackageConfig(JarInputStream jarStream, St
     /**
      * Load all the security weave packages embedded in the agent jar.
      */
-    private Collection<ClassMatchVisitorFactory> loadInternalSecurityWeavePackages() {
+    private void loadInternalSecurityWeavePackages() {
```

Comment:
This method returned a collection, which was instantiated and never populated.
Also the return value was always ignored. So removing the allocation and the return value.